tests/playwright/Report.md

```diff
@@ -0,0 +1,213 @@
+# Evaluation of Microsoft Playwright
+
+## Table of Contents
+
+[Introduction](#Introduction)
+
+[Project Background](#project-background)
+
+[Playwright Vs WebdriverIO](#playwright-vs-webdriverIO)
+
+* [Sample Playwright Page screenshot code](#sample-playwright-page-screenshot-code)
+* [Playwright and Jest](#playwright-and-jest)
```

Comment:
I would recommend simplifying the titles. Since this section already talks about playwright, it would make sense to simplify the title to just `Jest`. Also, if you have Jest section while talking about playwright, it would make sense to have `Mocha` section when talking about webdriverio.

Reply:
Noted.